all-in-one/Dockerfile

```
@@ -66,19 +66,13 @@ RUN if [ -n "$REST_PATH" ]; \
     mv oscal-rest-service-app/target/oscal-rest-service-app*.jar "/app/oscal-rest-service-app.jar"; \
     fi
 
-FROM openjdk:${JAVA_VERSION}-slim-bullseye
+FROM gcr.io/distroless/java11-debian11
```

```suggestion
FROM gcr.io/distroless/java11-debian11:nonroot
```

> Note: this doesn't replace my other comment but I want to discuss both changes separately.

It looks like a `nonroot` tag is published. Does this do what I'd expect as far as user, etc? If so, can we use the tag? Or does it mean something else?